src/content/app/species/components/species-sidebar-modal/modal-views/SpeciesSidebarDownloads.tsx

```diff
@@ -26,53 +26,69 @@ import ExternalLink from 'src/shared/components/external-link/ExternalLink';
 import styles from './SpeciesSidebarDownloads.module.css';
 
 const SpeciesSidebarDownloads = () => {
+  //Fetch an array of ftp links and convert to object
   const activeGenomeId = useAppSelector(getActiveGenomeId) || '';
   const { data: fileLinksResponse } = useSpeciesFileLinksQuery(activeGenomeId, {
     skip: !activeGenomeId
   });
   const fileLinks = fileLinksResponse?.links || [];
-  const linkSections: Record<string, string> = {};
-
+  const fileLinkURLs: Record<string, string> = {};
   for (const link of fileLinks) {
-    linkSections[link.dataset_type] = link.path;
+    fileLinkURLs[link.dataset_type] = link.url;
   }
-
+  //Annotations from Andy
+  type LinkAnnotation = {
+    type: string;
+    title: string;
+    text: string;
+  };
+  type LinkSection = {
+    title: string;
+    links: LinkAnnotation[];
+  };
+  const linkAnnotations: LinkSection[] = [
+    {
+      title: 'Assembly',
+      links: [{ type: 'assembly', title: 'DNA sequence', text: 'FASTA' }]
+    },
+    {
+      title: 'Annotation',
+      links: [
+        {
+          type: 'genebuild',
+          title: 'Genes, cDNA, ncRNA, proteins',
+          text: 'FASTA/GTF/GFF3'
+        },
+        { type: 'variation', title: 'Variation', text: 'VCF' },
+        { type: 'homologies', title: 'Homologies', text: 'TSV' },
+        {
+          type: 'regulation',
+          title: 'Regulation',
+          text: 'BigBED/TSV/GFF3/BigWig'
+        }
+      ]
+    }
+  ];
+  //Build the content
   return (
     <div>
-      {linkSections.assembly && (
-        <section>
-          <div className={styles.sectionHead}>Assembly</div>
-          <SpeciesFilesLink title="DNA sequence" link={linkSections.assembly}>
-            FASTA
-          </SpeciesFilesLink>
+      {linkAnnotations.map((linkSection) => (
+        <section key={linkSection.title}>
+          <div className={styles.sectionHead}>{linkSection.title}</div>
+          {linkSection.links.map(
+            (link) =>
+              fileLinkURLs[link.type] && (
+                <SpeciesFilesLink
+                  title={link.title}
+                  link={fileLinkURLs[link.type]}
+                  key={link.type}
+                >
+                  {link.text}
+                </SpeciesFilesLink>
+              )
+          )}
```

Comment:
As a side note; while the code after the refactoring has become smarter; I am not sure it has become more compact or easier to read.

Reply:
Would reverting the last comment make it better ot worse?

Reply:
I think it read slightly easier before the refactor (more declarative, less need to jump between different sections of the code to see what's happening); but I don't think one is significantly better or worse than the other. I am ok with either.

Reply:
Refactor reverted